### PR TITLE
sd-path: some fixups

### PR DESCRIPTION
**Pull request by poettering**
If we make this public now, let's make sure to stick to a single naming scheme, and make sure we actually export the correct data.

### CODE REVIEW

File: src/libsystemd/sd-path/sd-path.c [hunk 1]
```diff
@@ -321,23 +321,23 @@ static int get_path(uint64_t type, char **buffer, const char **ret) {
                 return from_user_dir("XDG_DESKTOP_DIR", buffer, ret);
 
         case SD_PATH_SYSTEMD_UTIL_DIR:
-                *ret = ROOTPREFIX "lib/systemd";
+                *ret = ROOTPREFIX "/lib/systemd";
```
Comment (keszybz) [inline src/libsystemd/sd-path/sd-path.c:324]: Indeed, this shows (before your patch):
```console
$ build/systemd-path systemd-util-dir
/usrlib/systemd
```
I looked at the output many times when writing this, but somehow I didn't see this one.

File: src/core/systemd.pc.in [hunk 2]
```diff
@@ -65,16 +65,16 @@ systemdshutdowndir=${systemd_shutdown_dir}
...
-sysctl_dir=${prefix}/lib/sysctl.d
+sysctl_dir=${rootprefix}/lib/sysctl.d
 sysctldir=${sysctl_dir}
 
-binfmt_dir=${prefix}/lib/binfmt.d
+binfmt_dir=${rootprefix}/lib/binfmt.d
 binfmtdir=${binfmt_dir}
 
-modules_load_dir=${prefix}/lib/modules-load.d
+modules_load_dir=${rootprefix}/lib/modules-load.d
```
Comment (keszybz) [inline src/core/systemd.pc.in:77]: All I can say is "meh, who cares" ;).

File: man/sd_path_lookup.xml [hunk 3]
```diff
@@ -76,30 +76,30 @@
...
+        <constant>SD_PATH_SYSTEMD_SYSTEM_PRESET</constant>,
+        <constant>SD_PATH_SYSTEMD_USER_UNIT</constant>,
+        <constant>SD_PATH_SYSTEMD_USER_PRESET</constant>,
+        <constant>SD_PATH_SYSTEMD_SYSTEM_CONF</constant>,
+        <constant>SD_PATH_SYSTEMD_USER_CONF</constant>,
+        <constant>SD_PATH_SYSTEMD_SEARCH_SYSTEM_UNIT</constant>,
+        <constant>SD_PATH_SYSTEMD_SEARCH_USER_UNIT</constant>,
+        <constant>SD_PATH_SYSTEMD_SYSTEM_GENERATOR</constant>,
+        <constant>SD_PATH_SYSTEMD_USER_GENERATOR</constant>,
+        <constant>SD_PATH_SYSTEMD_SEARC_SYSTEM_GENERATOR</constant>,
```
Comment (keszybz) [inline man/sd_path_lookup.xml]: Typo here.

**Comment by poettering** [comment]
new version force pushed, only change is the suggested one.

**Pull request merged**